inc/command/class-command.php

```diff
@@ -213,11 +214,18 @@ protected function phpunit( InputInterface $input, OutputInterface $output ) {
 	 */
 	protected function codecept( InputInterface $input, OutputInterface $output ) {
 		$options = $input->getArgument( 'options' );
-		$tests_folder = $input->getOption( 'path' );
+		// Codeception command runs within `vendor` directory, so paths are relative to that.
+		$tests_folder = '../' . $input->getOption( 'path' );
 		$output_folder = $input->getOption( 'output' );
-		$run_headless_browser = $input->getOption( 'browser' );
+		$run_headless_browser = $input->getOption( 'browser' ) ?: 'chrome';
 		$use_chassis = $input->getOption( 'chassis' );
 		$project_subdomain = $this->get_project_subdomain();
+		$subsubcommand = $input->getArguments()['options'][0] ?? '';
+		$test_suite = $this->get_test_suite_argument( $input );
+
+		if ( $subsubcommand === 'bootstrap' ) {
```

Comment:
Do we need to consider anything in relation to codecept's built in bootstrap command? This basically does the same thing but more tailored to our use case so not a major issue, just wanting to make a note.

Reply:
Nothing that comes to mind, we can always revisit if something needs to be added.